main.rb

```diff
@@ -0,0 +1,93 @@
+
+module Enumerable
+    def my_each
+        for item in self
+            yield(item)
+        end
+    end
```

Comment:
- [ ] should have working `my_each` identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
- Your method is not returning an enumerator when no block is given.